pkgs/applications/audio/samplebrain/default.nix

```
@@ -0,0 +1,56 @@
+{
+  fetchFromGitLab,
+  fftw,
+  lib,
+  liblo,
+  libsndfile,
+  makeDesktopItem,
+  mkDerivation,
+  portaudio,
+  qmake,
+  qtbase,
+  wrapQtAppsHook,
+}:
+mkDerivation rec {
+  pname = "samplebrain";
+  version = "0.18";
+  src = fetchFromGitLab {
+    owner = "then-try-this";
+    repo = "samplebrain";
+    rev = "${version}_release";
+    sha256 = "sha256-k5VFaIo6XKv9n6OQNNDLVneKQDBZA4eFu0RNea9DpVM=";
+  };
+  nativeBuildInputs = [
+    qmake
+    wrapQtAppsHook
+  ];
+  buildInputs = [
+    fftw
+    liblo
+    libsndfile
+    portaudio
+    qtbase
+  ];
```

```suggestion
  version = "0.18";

  src = fetchFromGitLab {
    owner = "then-try-this";
    repo = "samplebrain";
    rev = "${version}_release";
    sha256 = "sha256-k5VFaIo6XKv9n6OQNNDLVneKQDBZA4eFu0RNea9DpVM=";
  };

  nativeBuildInputs = [
    qmake
    wrapQtAppsHook
  ];

  buildInputs = [
    fftw
    liblo
    libsndfile
    portaudio
    qtbase
  ];

```
Having some empty lines makes the code more readable and less one giant text block.